blogposts/2022/release-3.39

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Sourcegraph 3.39 release'
+publishDate: 2022-04-21T10:00-07:00
+description: 'Sourcegraph 3.39 introduces...'
+tags: [blog, release]
+slug: 'release/3.39'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-39-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-39-release.png
+changelogItems:
+
+---
+
+Sourcegraph 3.39 is now available! In this release:
+
+## Create living documentation with Notebooks, now Generally Available
```

Comment:
@benvenker Could you take a shot at this section please? Do you have any visuals to include for Notebooks?

Reply:
Yep, taking a look now. Plenty of visuals to choose from, or I can create a new one if you had something specific in mind (GIF/video of something vs. an image, or a particular bit of functionality).